CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to cofidectl
+
+Linting, unit tests and integration tests are run by GitHub Actions against changes proposed to `cofidectl`.
```

Comment:
Missing an intro that contributions are accepted through PRs that need to be reviewed by the Cofide team and pass checks etc. (Also could mention that Google Code Assist is doing reviews to save contributors from heart attacks ;-) ).